test/test_functional.py

```diff
@@ -0,0 +1,133 @@
+            'normalized': True,
+            'onesided': True,
+        }
+
+        for i in range(self.number_of_trials):
+            test_data = torch.rand(self.data_sizes)
+            self._test_istft_helper(test_data, kwargs1)
+
+    def test_istft2(self):
+        # hann_window, centered, not normalized, not onesided
+        kwargs2 = {
+            'n_fft': 12,
+            'hop_length': 2,
+            'win_length': 8,
+            'window': torch.hann_window(8),
+            'center': True,
+            'pad_mode': 'reflect',
+            'normalized': False,
+            'onesided': False,
+        }
+
+        for i in range(self.number_of_trials):
+            test_data = torch.rand(self.data_sizes)
+            self._test_istft_helper(test_data, kwargs2)
+
+    def test_istft3(self):
+        # hamming_window, centered, normalized, not onesided
+        kwargs3 = {
+            'n_fft': 15,
+            'hop_length': 3,
+            'win_length': 11,
+            'window': torch.hamming_window(11),
+            'center': True,
+            'pad_mode': 'constant',
+            'normalized': True,
+            'onesided': False,
+        }
+
+        for i in range(self.number_of_trials):
+            test_data = torch.rand(self.data_sizes)
+            self._test_istft_helper(test_data, kwargs3)
+
+    def test_istft4(self):
+        # hamming_window, not centered, not normalized, onesided
+        # window same size as n_fft
+        kwargs4 = {
+            'n_fft': 5,
+            'hop_length': 2,
+            'win_length': 5,
+            'window': torch.hamming_window(5),
+            'center': False,
+            'pad_mode': 'constant',
+            'normalized': False,
+            'onesided': True,
+        }
+
+        for i in range(self.number_of_trials):
+            test_data = torch.rand(self.data_sizes)
+            self._test_istft_helper(test_data, kwargs4)
+
+    def test_istft5(self):
+        # hamming_window, not centered, not normalized, not onesided
+        # window same size as n_fft
+        kwargs5 = {
+            'n_fft': 3,
+            'hop_length': 2,
+            'win_length': 3,
+            'window': torch.hamming_window(3),
+            'center': False,
+            'pad_mode': 'reflect',
+            'normalized': False,
+            'onesided': False,
+        }
+
+        for i in range(self.number_of_trials):
+            test_data = torch.rand(self.data_sizes)
+            self._test_istft_helper(test_data, kwargs5)
+
+    def test_istft6(self):
+        # stft = torch.stft(torch.ones(4), 4)
+        stft = torch.tensor([
+            [[4., 0.], [4., 0.], [4., 0.], [4., 0.], [4., 0.]],
+            [[0., 0.], [0., 0.], [0., 0.], [0., 0.], [0., 0.]],
+            [[0., 0.], [0., 0.], [0., 0.], [0., 0.], [0., 0.]]
+        ])
+
+        estimate = torchaudio.functional.istft(stft, n_fft=4, length=4)
+        self._compare_estimate(torch.ones(4), estimate)
+
+    def test_istft7(self):
```

Comment:
The goal would be clearer with a name like `test_istft_of_zero`.

Reply:
Done.